config/base.js

```diff
@@ -65,9 +65,10 @@ const SCHEMA_SUBSCRIBE = Joi.object().keys({
 
 const SCHEMA_CONFIG = Joi.object()
     .keys({
+        template: Joi.string().regex(Regex.FULL_TEMPLATE_NAME_WITH_NAMESPACE),
         version: Joi.number().integer().min(1).max(50),
         annotations: Annotations.annotations,
-        jobs: SCHEMA_JOBS.required(),
+        jobs: SCHEMA_JOBS.optional(),
```

Comment:
should we be making this optional or make a separate schema config for pipeline template?

Reply:
pipeline that end up with no jobs will still be caught by config-parser, unless there's something else that uses this schema I would say we don't have to?

Reply:
We should conditionally make `jobs` as `required` in the schema when `template` is not specified